pkg/skelington/Library/Skeleton.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+({
+template: html`
+<style>
+  :host {
+    color: var(--theme-color-fg-1);
+    background-color: var(--theme-color-bg-0);
+    ${globalThis.themeRules}
+  }
+</style>
+
+<h2 center>ARCS   SKELETON   SAYS   HELLO</h2>
```

Comment:
`skelington`? 🤣 
(nit - is spelled with 2 L i think...)

Reply:
🙏

Reply:
Yes, it should probably have two ELLs, for the record. I'm only invoking artistic license. :)